Sources/RichTextKit/Component/RichTextViewComponent+Attributes.swift

```diff
@@ -15,14 +15,15 @@ public extension RichTextViewComponent {
         if hasSelectedRange {
             return richTextAttributes(at: selectedRange)
         } else {
-            #if macOS
+            return typingAttributes
+           // #if macOS
             // TODO: If is link or any mention, set range to location and 0.
             let range = NSRange(location: selectedRange.location - 1, length: 1)
             let safeRange = safeRange(for: range)
             return richTextAttributes(at: safeRange)
-            #else
+         //   #else
             return typingAttributes
-            #endif
+          //  #endif
```

Comment:
Self exploration: 

This is needed because on macOS when we set font or fontSize, we don't want to send the typing attributes right away. Instead, we want to get `fontSizeAtCurrentPosition` as `setFontSizeAtCurrentPosition` method describes.